documentation/enforce/README.md

```diff
@@ -9,6 +9,48 @@ Passable('enforcement', (pass, enforce) => {
         enforce('str1').allOf({...}).anyOf({...});
         enforce('str2').noneOf({...});
         enforce('str3').anyOf({...});
+        enforce(4).isNumber();
     });
 });
-```
\ No newline at end of file
+```
+
+## Chaining enforce functions
+
+All of these functions are chainable. All functions other can be used more than once.
```

Comment:
I'm not sure I understand the second part of this sentence.

Reply:
Me neither 😕